src/workflow/CompareCommitToRevisionWorkflow.php

```diff
@@ -354,8 +373,15 @@ private function normalizeDiff($text) {
     // matches. A "context line" in a patch is a line that starts with a space.
     $val = preg_replace('/^ .*$/m', '', $val);
 
-    // sometimes, the patches differ only by copy or rename.  Normlize them.
+    // sometimes, the patches differ only by copy or rename.  Normalize them.
     $val = preg_replace('/^(copy|rename)(.*)$/m', 'copy-or-rename$2', $val);
+
+    // remove text that says "No newline at end of file"
+    $val = preg_replace('/^\\\ No newline at end of file$/m', '', $val);
+
+    // strip out all blank lines
+    $val = trim(preg_replace("/(^[\r\n]|[\r\n]+)/", "\n", $val));
```

Comment:
Can we simplify a bit but produce the same result 
```suggestion
    $val = preg_replace("/[\r\n]+/", "\n", trim($val));
```

Reply:
yeah, good feedback.  Fixed this.  Thanks!